.kuzzlerc.sample

```diff
@@ -212,7 +212,111 @@
+    //    coming from it
+    // Entry points can be combined.
+    "entryPoints": {
+      "embedded": true,
+      "proxy": false
+    },
+
+    // [proxy]
+    // A Kuzzle proxy can be used to forward requests to Kuzzle.
+    //   * host:
+    //       Host name/address of a Kuzzle proxy server.
+    //       Can be a hostname, an IP address or an URI.
+    //   * pingTimeout:
+    //       Timeout (in milliseconds) of a ping request. If reached, Kuzzle
+    //       marks its connection to the distant host as lost.
+    //   * pingInterval:
+    //       Time (in milliseconds) between ping requests
+    //   * port:
+    //       Network port opened by the Kuzzle proxy server
+    //   * retryClientListDelay
+    //       Time after which (in milliseconds) Kuzzle should ask the proxy to
+    //       send the list of clients back after a disconnection (default 1s).
+    //   * retryInterval:
+    //       Time interval (in milliseconds) between reconnection retries
+    //       when the connection to the broker server has been lost
+    "proxy": {
+      "host": "localhost",
+      "port": 7331,
+      "resendClientListDelay": 1000,
+      "retryInterval": 1000
+    },
+
+    // [logs] (embedded only)
+    // Configuration section for Kuzzle access logs
+    //   * transports:
+    //       An array of Winston transports configurations to output access
+    //       logs. Possible transport types are: console, file, elasticsearch
+    //       and syslog.
+    //       Please refer to https://github.com/winstonjs/winston/blob/master/docs/transports.md
+    //       for more information on transports configuration.
+    //   * accessLogFormat:
+    //       Access log format.
+    //       Currently supported are "combined" (=Apache combined logs format) and
+    //       "logstash".
+    //       "logstash" will output the whole request input to JSON, ready to
+    //       be consumed by logstash agent.
+    //   * accessLogIpOffset:
+    //       The offset to use as the client ip, from the FORWARDED-FOR chain,
+    //       beginning from the right (0 = the ip address of the last
+    //       client|proxy which connected to Kuzzle
+    "logs": {
+      "transports": [
+        {
+          "transport": "console",
+          "level": "info",
+          "stderrLevels": []
+        }
+      ],
+      "accessLogFormat": "combined",
+      "accessLogIpOffset": 0
+    },
+    // (embedded only)
+    //   * maxRequestSize:
+    //     The maxium size of an incoming request. Units can be expressed in
+    //     bytes ("b" or none), kilobytes ("kb"), megabytes ("mb"), gigabytes
+    //     ("gb") or terabytes ("tb")
+    //   * port:
+    //     The port on which Kuzzle should bind to
+    //
+    "maxRequestSize": "1mb",
+    "port": 7512,
+    // [protocols] (embedded only)
+    // protocols accepted by Kuzzle.
+    // protocols can be extended using plugins and configured in this section.
+    "protocols": {
+      // * maxFormFileSize:
```

Comment:
nitpicking: move the comment just above the related entry would be more readable:
```
"protocols": {
  "http": {
    "enable": true,
    // maximum size of documents sent via http forms:
    "maxFormFileSize": "1mb"
  },
  "socketio": {
    "enable": true,
    // value of Access-Control-Allow-Origin header to answer the upgrade request:
    "origins": '*:*' 
  },
  "websocket": {
    "enable": true
  }
}
```